mappings/net/minecraft/entity/mob/PhantomEntity.mapping

```diff
@@ -1,12 +1,15 @@
 CLASS net/minecraft/unmapped/C_wbqjthgx net/minecraft/entity/mob/PhantomEntity
+	FIELD f_cnhcyzzr FLAP_DEGREES_PER_TICK F
```

Comment:
What exactly does this describe? If it's dive angle wouldn't `FALP_DIVE_ANGLE` Make more sense? This name feels really weird IMO

Reply:
I copied this one from Mojmap. As I understood, it's used by the renderer/model to determine the wings angle from a tick offset and animation progress (see code for `PhantomEntityModel#setAngles`)